src/Validators/StringValidator.php

```diff
@@ -6,6 +6,7 @@ class StringValidator
 {
     public const MAXVARCHARLENGTH = 255;
     public const MAXTEXTLENGTH = 10000;
+    private const PATTERN = "/^[a-z ,.'-]+$/i";
```

Comment:
not a great name for a constant given the generic nature of this validator

Reply:
fixed